blockchains/erc20/erc20_worker.js

```diff
@@ -63,17 +99,23 @@ class ERC20Worker extends BaseWorker {
 
     logger.info(`Fetching transfer events for interval ${result.fromBlock}:${result.toBlock}`);
 
-    let events;
+    let events = [];
     let overwritten_events = [];
     if ('extract_exact_overwrite' === constants.CONTRACT_MODE) {
-      events = await contractEditor.getPastEventsExactContracts(this.web3, result.fromBlock, result.toBlock,
-        new TimestampsCache());
-      contractEditor.changeContractAddresses(events);
+      const timestampsCache = new TimestampsCache();
```

Comment:
Couldn't we just add this `new TimestampCache()` before the if-else statements and give it to the functions?